tests/xharness/Jenkins.cs

```diff
@@ -219,8 +224,19 @@ bool IsIncluded (TestProject project)
 		{
 			if (!project.IsExecutableProject)
 				return false;
+			
+			if (project.IsBclTest) {
+				if (!project.IsNewBclTest)
+					return IncludeBcl || IncludeOldBCL;
+				if (project.IsMscorlib) 
+					return IncludeMscorlib;
+				return IncludeBcl || IncludeNewBCL;
```

Comment:
This means the mscorlib tests are not included when only `IncludeBcl` is true. Is that expected?

Reply:
Yes, I wanted the mscorlib tests to be included explicitly.